test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -0,0 +1,294 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test for randomized benchmarking experiments."""
+
+from test.base import QiskitExperimentsTestCase
+
+import numpy as np
+from ddt import ddt, data, unpack
+from qiskit.circuit.library import SXGate, CXGate, TGate, XGate
+from qiskit.exceptions import QiskitError
+from qiskit.providers.aer import AerSimulator
+from qiskit.providers.aer.noise import NoiseModel, depolarizing_error
+from qiskit.quantum_info import Clifford
+from qiskit.quantum_info.operators.predicates import matrix_equal
+
+from qiskit_experiments.library import randomized_benchmarking as rb
+
+
+class RBTestCase(QiskitExperimentsTestCase):
+    """Base test case for randomized benchmarking defining a common noise model."""
+
+    def setUp(self):
+        """Setup the tests."""
+        super().setUp()
+
+        # depolarizing error
+        self.p1q = 0.002
```

Comment:
If you increase the error rates so you could run shorter RB sequences to speed up test time